tensorrt_llm/_torch/modules/mhc/mhc_cuda.py

```
@@ -698,48 +698,32 @@ def _fused_hc_call(
-    __slots__ = ("n", "hidden_size", "_cache", "_maxsize")
+    __slots__ = ("n", "hidden_size", "_slot_key", "_slot_val")
 
-    # Up to 48 distinct entries — covers the 35 CUDA-graph decode buckets
-    # plus headroom. Each entry at B<=256 is under ~10 MB.
-    DEFAULT_MAXSIZE = 48
-    # Skip the cache above this B; prefill rides the torch allocator.
+    # Skip the slot above this B; prefill rides the torch allocator.
     _CACHE_MAX_B = 256
 
-    def __init__(self, n: int, hidden_size: int, maxsize: int = DEFAULT_MAXSIZE):
+    def __init__(self, n: int, hidden_size: int, maxsize: int = None):
```

Ruff (0.15.20)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)